From 0ed5ab73b2da7e1695df741e51ce3ddaf8ea75b9 Mon Sep 17 00:00:00 2001
From: Helmut Grohne <helmut@subdivi.de>
Date: Wed, 20 Jan 2016 23:58:23 +0100
Subject: [PATCH] cross cmake should specify existing compilers

It seems that cross symlinks for $triplet-cc and $triplet-c++ are no
longer generated in the gcc-defaults. Thus switch to the canonical gcc
compiler names.
---
 Debian/Debhelper/Buildsystem/cmake.pm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Debian/Debhelper/Buildsystem/cmake.pm b/Debian/Debhelper/Buildsystem/cmake.pm
index f4d5c3c..98631e7 100644
--- a/Debian/Debhelper/Buildsystem/cmake.pm
+++ b/Debian/Debhelper/Buildsystem/cmake.pm
@@ -67,12 +67,12 @@ sub configure {
 		if ($ENV{CC}) {
 			push @flags, "-DCMAKE_C_COMPILER=" . $ENV{CC};
 		} else {
-			push @flags, "-DCMAKE_C_COMPILER=" . dpkg_architecture_value("DEB_HOST_GNU_TYPE") . "-cc";
+			push @flags, "-DCMAKE_C_COMPILER=" . dpkg_architecture_value("DEB_HOST_GNU_TYPE") . "-gcc";
 		}
 		if ($ENV{CXX}) {
 			push @flags, "-DCMAKE_CXX_COMPILER=" . $ENV{CXX};
 		} else {
-			push @flags, "-DCMAKE_CXX_COMPILER=" . dpkg_architecture_value("DEB_HOST_GNU_TYPE") . "-c++";
+			push @flags, "-DCMAKE_CXX_COMPILER=" . dpkg_architecture_value("DEB_HOST_GNU_TYPE") . "-g++";
 		}
 	}
 
-- 
2.7.0.rc3

